frontends/ol-components/src/components/TiptapEditor/extensions/node/Quote/Quote.ts

```diff
@@ -0,0 +1,17 @@
+import Blockquote from "@tiptap/extension-blockquote"
+
+export const Quote = Blockquote.extend({
+  name: "quote",
+
+  parseHTML() {
+    return [
+      { tag: "quote" }, // Custom tag
```

Comment:
General comment: Let's stick to regular / valid HTML tags unless there's a very good reason.